hooks/auth/useSignUpMutation.ts

```
@@ -10,12 +13,17 @@ interface SignUpFormData {
 }
 
 const signUpUser = async (formData: SignUpFormData) => {
-  const response = await api.post('/4/user', formData);
-  return response.data;
+  try {
+    const response = await api.post('/4/user', formData);
+    return response.data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)